.github/workflows/lint.yml

```diff
@@ -0,0 +1,24 @@
+name: Sanitize
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    name: "Lint"
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+    steps:
```

Comment:
Maybe instead use their Action: https://black.readthedocs.io/en/stable/integrations/github_actions.html

Also, maybe make this a monthly job and have this Action check on each PR: https://github.com/marketplace/actions/python-black-formatter-only-on-edited-files
It only check changed files, so it won't put people on the hook for code unrelated to what they are changing.

Reply:
Even better/worse: have a bot autopush blacked files to a PR: https://github.com/psf/black/blob/2234639a31ed8aeca05e20aa6966005277e9ae07/.github/workflows/autoblack.yml

Reply:
Using their action is great though.

Reply:
You're right: by merging this PR there should be no way black will find errors in other areas but those touched in future PRs.

Having the bot autocorrect is neat though, and scary. What do you think?